src/interfaces.ts

```diff
@@ -123,10 +118,25 @@ export interface Connection {
   contract: string;
   list: () => Promise<TableMetadata[]>;
   create: (
-    query: string,
-    options?: CreateTableOptions
+    /** The ChainId of the chain this table should be created on.
+     *  MUST match the chain that the SDK connected to
+     **/
+    chainId: number,
```

Comment:
I see that this is required. Could we don't default to whatever chained the sdk is connected to in the public APIs? This information can be pulled from the ethers connection.

Reply:
I was thinking the same thing, but the TPD docs mention making this required.  Maybe I was reading the intent wrong?  We can absolutely remove this argument all together.  The only reasons to include it afaik is so the caller is forced to be explicit, and maybe in the future calls to create could be direct to the SC without any validator connection.
Honestly I'd be happy removing it.
here is the Notion doc that I was referencing: https://www.notion.so/textile/Align-the-intention-more-directly-with-the-outcome-when-creating-tables-a82b38c7606942e48f9b3f6a033f516f#e1461d1d49574c92ab00d9d3e2674840

Reply:
I think in the TPD doc, the _explicit_ talk was more about the APIs on the validators (though I see it _is_ included in the mention of the SDK). In any case, the SDKs should paper over this kind of thing. Even with calls direct to the smart contract, the SDK could provide a default.

Reply:
If you search through the resolved comments there, we mention something about the SDK letting the user avoid adding chain id and other things that are always fixed.